cdap-ui/app/directives/breadcrumbs/breadcrumbs.less

```diff
@@ -114,6 +71,52 @@ my-breadcrumbs, my-breadcrumb {
       }
     }
 
+    ul.dropdown-menu {
+      width: 120px;
+      height: auto;
+      max-height: 290px;
+      min-width: 0;
+      background-color: white;
+      border-color: white;
+      padding: 10px 0;
+      margin-top: 1px;
+      .border-radius(4px);
+      .box-shadow(0 6px 12px rgba(0, 0, 0, 0.175));
+      overflow-x: hidden;
+      overflow-y: scroll;
+      @media (max-width: @screen-md-max) {
+        position: absolute;
+      }
+      > li {
+        background-color: white;
+        padding: 0;
+        &:focus {
+          outline: 0;
+        }
+        > a {
+          color: @cdap-header;
+          font-weight: 500;
+          background-color: white;
+          padding-right: 10px;
+          padding-left: 10px;
+          &:hover, &:focus {
+            color: @cdap-orange;
+          }
+          &:focus {
+            outline: 0;
+          }
+        }
+        i {
+          display: inline-block;
+          width: 13px;
+        }
+        .active {
+          color: #c4c4c4;
+          pointer-events: none;
+        }
+      }
+    }
+
     .breadcrumb {
```

Comment:
This might be out of context of this PR, but .breadcrumb style shouldn't it be under ```breadcrumb``` directive?